ts/features/wallet/creditCard/screen/CreditCardDetailScreen.tsx

```diff
@@ -10,14 +10,14 @@ import BasePaymentMethodScreen from "../../common/BasePaymentMethodScreen";
 import PaymentMethodFeatures from "../../component/features/PaymentMethodFeatures";
 import CreditCardComponent from "../component/CreditCardComponent";
 
-type NavigationParams = Readonly<{
+export type CreditCardDetailScreenNavigationParams = Readonly<{
```

Comment:
🤓 **suggestion**: I see we do have a tendency to be overly descriptive. Given that the file name is already `CreditCardDetailScreen` why don't we just keep `NavigationParams`? I see no real need for context that is already there.

Reply:
I see your point, but I think in this case the extra description of the type could be worth, since it is not only for internal use, but must also be exported externally. 
For example, if we consider the type `WalletParamsList` in `ts/navigation/params/WalletParamsList.ts`, it must describe *all* the navigation params for the navigator. If we would use only the generic `NavigationParams` for each of these screens, we should use a lot of aliases in the import phase in order to distinguish the different screens, also with the possibility to associate one of the `NavigationParams` to the wrong screen. With an explicit screen there is no possibility to ambiguity when import a `NavigationParams` for a specific screen and it is also more easy with the help of the IDE auto-import.
What you think about that?

Reply:
I'm more leaning toward simplicity tbh: I'd rather have many aliased imports in one place to highlight the tight coupling. But your point also makes sense and I think we are in the realm of opinions 😄 therefore... let's keep the _status quo_.